2021-02-08 21:06:42,804 INFO    MainThread:2296 [wandb_setup.py:_flush():70] setting env: {}
2021-02-08 21:06:42,805 INFO    MainThread:2296 [wandb_setup.py:_flush():70] multiprocessing start_methods=spawn
2021-02-08 21:06:42,805 INFO    MainThread:2296 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-08 21:06:42,805 INFO    MainThread:2296 [wandb_init.py:_log_setup():303] Logging user logs to C:\Users\Alikhan\OneDrive - Tonbridge School\School\EPQ\SMP Challenge\IIPA-Flickr-Dataset-Implementation\wandb\run-20210208_210642-2zsv90kr\logs\debug.log
2021-02-08 21:06:42,806 INFO    MainThread:2296 [wandb_init.py:_log_setup():304] Logging internal logs to C:\Users\Alikhan\OneDrive - Tonbridge School\School\EPQ\SMP Challenge\IIPA-Flickr-Dataset-Implementation\wandb\run-20210208_210642-2zsv90kr\logs\debug-internal.log
2021-02-08 21:06:45,749 INFO    MainThread:2296 [wandb_run.py:_console_start():1237] atexit reg
2021-02-08 21:06:45,751 INFO    MainThread:2296 [wandb_run.py:_redirect():1104] redirect: SettingsConsole.WRAP
2021-02-08 21:06:45,751 INFO    MainThread:2296 [wandb_run.py:_redirect():1135] Wrapping output streams.
2021-02-08 21:06:45,751 INFO    MainThread:2296 [wandb_run.py:_redirect():1151] Redirects installed.
2021-02-08 21:06:45,751 INFO    MainThread:2296 [wandb_config.py:__setitem__():135] config set batch_size = 16 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000015C49BDB100>>
2021-02-08 21:06:45,752 INFO    MainThread:2296 [wandb_run.py:_config_callback():549] config_cb batch_size 16 {'_wandb': {'cli_version': '0.10.12', 'python_version': '3.8.5', 'is_jupyter_run': False, 'is_kaggle_kernel': False, 'framework': 'torch'}, 'batch_size': 16}
2021-02-08 21:06:45,752 INFO    MainThread:2296 [wandb_config.py:__setitem__():135] config set validation_percentage = 0.2 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000015C49BDB100>>
2021-02-08 21:06:45,752 INFO    MainThread:2296 [wandb_run.py:_config_callback():549] config_cb validation_percentage 0.2 {'_wandb': {'cli_version': '0.10.12', 'python_version': '3.8.5', 'is_jupyter_run': False, 'is_kaggle_kernel': False, 'framework': 'torch'}, 'batch_size': 16, 'validation_percentage': 0.2}
2021-02-08 21:06:45,753 INFO    MainThread:2296 [wandb_config.py:__setitem__():135] config set lr = 0.01 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000015C49BDB100>>
2021-02-08 21:06:45,753 INFO    MainThread:2296 [wandb_run.py:_config_callback():549] config_cb lr 0.01 {'_wandb': {'cli_version': '0.10.12', 'python_version': '3.8.5', 'is_jupyter_run': False, 'is_kaggle_kernel': False, 'framework': 'torch'}, 'batch_size': 16, 'validation_percentage': 0.2, 'lr': 0.01}
2021-02-08 21:06:45,754 INFO    MainThread:2296 [wandb_config.py:__setitem__():135] config set random_seed = 99 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000015C49BDB100>>
2021-02-08 21:06:45,754 INFO    MainThread:2296 [wandb_run.py:_config_callback():549] config_cb random_seed 99 {'_wandb': {'cli_version': '0.10.12', 'python_version': '3.8.5', 'is_jupyter_run': False, 'is_kaggle_kernel': False, 'framework': 'torch'}, 'batch_size': 16, 'validation_percentage': 0.2, 'lr': 0.01, 'random_seed': 99}
2021-02-08 21:06:45,755 INFO    MainThread:2296 [wandb_config.py:__setitem__():135] config set freeze = False - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000015C49BDB100>>
2021-02-08 21:06:45,755 INFO    MainThread:2296 [wandb_run.py:_config_callback():549] config_cb freeze False {'_wandb': {'cli_version': '0.10.12', 'python_version': '3.8.5', 'is_jupyter_run': False, 'is_kaggle_kernel': False, 'framework': 'torch'}, 'batch_size': 16, 'validation_percentage': 0.2, 'lr': 0.01, 'random_seed': 99, 'freeze': False}
2021-02-08 21:06:49,426 INFO    MainThread:2296 [wandb_run.py:_atexit_cleanup():1207] got exitcode: 0
2021-02-08 21:06:49,426 INFO    MainThread:2296 [wandb_run.py:_restore():1179] restore
2021-02-08 21:06:49,432 INFO    MainThread:2296 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "1fbcf59c9e6644f2952dd1264a8372a7"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 621
      total_bytes: 621
    }
  }
}

2021-02-08 21:06:51,443 INFO    MainThread:2296 [wandb_run.py:_wait_for_finish():1324] got exit ret: uuid: "3f4e3e6cb0a944afb62c69a1091075f7"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 4217
      total_bytes: 4217
    }
  }
}

2021-02-08 21:06:53,187 INFO    MainThread:2296 [wandb_run.py:_show_files():1536] logging synced files
2021-02-08 21:06:53,198 INFO    MainThread:2296 [internal.py:handle_exit():138] Internal process exited
